PlayerControls/sources/DefaultControlsViewController.swift

```diff
@@ -110,6 +111,14 @@ public final class DefaultControlsViewController: ContentControlsViewController
     var task: URLSessionDataTask?
     private var animationsDuration: CFTimeInterval = 0.4
     
+    var shouldHideHomeIndicator = true
+    
+    @available(iOS 11.0, *)
+    override public func prefersHomeIndicatorAutoHidden() -> Bool
+    {
+        return shouldHideHomeIndicator
```

Comment:
Why do we need this? And why it is `true`?

Reply:
@AndriiMoskvin The method hides `homeIndicator` (the one that replaced Home button for iPhone X). So now when controls are hidden, that indicator becomes hidden too. 
You can watch this behavior on the gif image above.

Reply:
Than it should be part of the public props interface - because otherwise we are removing control of this from our clients.

Reply:
Implemented behavior is fine - I think it just needs to be overridable by app devs.

Reply:
I thought it is the default behavior, as in other players, including the default AVPlayer. Are you sure it is necessary to make it public? If yes, where does it have to be controlled from?

Reply:
Got it!